api/vudl.ini.dist

```diff
@@ -333,4 +333,23 @@ name = "Attribution-NonCommercial-NoDerivs"
 uri = "http://creativecommons.org/licenses/by-nc-nd/4.0/"
 [licenses.PUBLIC_DOMAIN]
 name = "CC0 1.0 Universal (Public Domain)"
-uri = "https://creativecommons.org/publicdomain/zero/1.0/"
\ No newline at end of file
+uri = "https://creativecommons.org/publicdomain/zero/1.0/"
+
+# Agent
+[agent]
+
+roles[] = "EDITOR"
+roles[] = "ARCHIVIST"
+roles[] = "PRESERVATION"
+roles[] = "DISSEMINATOR"
+roles[] = "CUSTODIAN"
+roles[] = "IPOWNER"
+roles[] = "CREATOR"
+
+types[] = "ORGANIZATION"
+types[] = "INDIVIDUAL"
+
+[agent.defaults]
+name = "Falvey Memorial Library, Villanova University"
```

Comment:
We should change this example to something like "My Library, My University" just to keep it generic and consistent with the rest of the example configuration. (I thought that I actually made the edit here previously, but I think it may have gotten force-pushed out of existence).

Reply:
Made change.